internal/mailer/template.go

```diff
@@ -33,6 +33,14 @@ func encodeRedirectURL(referrerURL string) string {
 	return referrerURL
 }
 
+const (
+	magiclinkMailType   = "magiclink"
+	recoveryMailType    = "recovery"
+	inviteMailType      = "invite"
+	signupMailType      = "signup"
+	emailChangeMailType = "email_change"
```

Comment:
Generally these should be uppercase. Feel free to create a separate type, though string is enough.